target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4019-eap2200.dts

```diff
@@ -0,0 +1,410 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/clock/qcom,gcc-ipq4019.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+
+/ {
+	model = "EnGenius EAP2200";
+	compatible = "engenius,eap2200";
+
+	chosen {
+		bootargs-append = " root=/dev/ubiblock0_1";
+	};
+
+        memory {
+                device_type = "memory";
+                reg = <0x80000000 0x10000000>;
+        };
+
+	soc {
+		rng@22000 {
+			status = "okay";
+		};
+
+		mdio@90000 {
+			#size-cells = <1>;
+			status = "ok";
+		};
+
+		ess-psgmii@98000 {
+			status = "okay";
+		};
+
+
+		blsp_dma: dma@7884000 {
+			status = "okay";
+		};
+
+		serial@78af000 {
+			pinctrl-0 = <&serial_0_pins>;
+			pinctrl-names = "default";
+			status = "okay";
+		};
+
+		serial@78b0000 {
+			pinctrl-0 = <&serial_1_pins>;
+			pinctrl-names = "default";
+			status = "okay";
+		};
+
+		spi0: spi@78b5000 {
+			pinctrl-0 = <&spi_0_pins>;
+			pinctrl-names = "default";
+			status = "okay";
+			cs-gpios = <&tlmm 12 GPIO_ACTIVE_HIGH>;
+
+			m25p80@0 {
+				#address-cells = <1>;
+				#size-cells = <1>;
+				reg = <0>;
+				compatible = "n25q128a11";
+				linux,modalias = "m25p80", "n25q128a11";
+				spi-max-frequency = <24000000>;
+				use-default-sizes;
+				partitions {
+					compatible = "fixed-partitions";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					partition0@0 {
+						label = "0:SBL1";
+						reg = <0x00000000 0x00040000>;
+						read-only;
+					};
+					partition1@40000 {
+						label = "0:MIBIB";
+						reg = <0x00040000 0x00020000>;
+						read-only;
+					};
+					partition2@60000 {
+						label = "0:QSEE";
+						reg = <0x00060000 0x00060000>;
+						read-only;
+					};
+					partition3@c0000 {
+						label = "0:CDT";
+						reg = <0x000c0000 0x00010000>;
+						read-only;
+					};
+					partition4@d0000 {
+						label = "0:DDRPARAMS";
+						reg = <0x000d0000 0x00010000>;
+						read-only;
+					};
+					partition5@e0000 {
+						label = "0:APPSBLENV";
+						reg = <0x000e0000 0x00010000>;
+						read-only;
+					};
+					partition6@f0000 {
+						label = "0:APPSBL";
+						reg = <0x000f0000 0x00080000>;
+						read-only;
+					};
+					partition7@170000 {
+						label = "0:ART";
+						reg = <0x00170000 0x00010000>;
+						read-only;
+					};
+				};
+			};
+		};
+
+		i2c0: i2c@78b7000 {
```

Comment:
No need for this, use the existing label from ipq4019.dtsi

Reply:
Thank you for your patience